README.md

```diff
@@ -16,33 +16,32 @@ This accelerator allows customers to quickly deploy an integrated
 platform and immediately start extracting insights from customer or
 employees\' conversations.
 
-**KEY FEATURES**:
+**KEY FEATURES:**
 
 Azure Cognitive Search Service and Azure OpenAI Service drive huge cost
 saving in call center operations while improving call center efficiency
 & customer satisfaction. Post-call insights inform actions, leading to
 better customer experiences, increased satisfaction, and improved
 staffing allocation for maximum efficiency.
 
--   Feature 1
+- Conversation Summarization and Key Phrase Extraction: Summarize long conversations into a short paragraph and pull out key phrases that are relevant to the conversation.
+- Batch speech-to-text using Azure Speech: Transcribe large amounts of audio files asynchronously including speaker diarizationand is typically used in post-call analytics scenarios. Diarizationis the process of recognizing and separating speakers in mono channel audio data.
+- Sensitive information extraction and redaction: Identify, categorize, and redact sensitive information in conversation transcription.
+- Sentiment analysis and opinion mining: Analyze transcriptions and associate positive, neutral, or negative sentiment at the utterance and conversation-level.
 
--   Feature 2
-
--   Feature 3
-
--   Feature 4
-
-**Below are image stills from the accelerator**\
+**Below are image stills from the accelerator.**\
 \
 ![image](/images/readMe/image2.png)
 
+You can view a live demo of the accelerator [here](https://conversational-knowledge-mining.azurewebsites.net/).
```

Comment:
The demo link is really small and buried compared to it prominence in the original repo. Since the demo is driving so much interest in the repo, it seems we'd want to feature it prominently. Has design provided guidance on this?

Reply:
No. We can move it when they do.